docs/tech/aca.md

```diff
@@ -0,0 +1,119 @@
+# Attribute Certificate Authority
+
+To support Attribute Based Access Control (ABAC) the user has to pass a set of attributes during TCert creation, these attributes will be used during transaction execution or query to determine whether the user can or cannot execute a specific chaincode. A mechanism to validate the ownership of attributes is required in order to prove if the attributes passed by the user are correct. The Attribute Certificate Authority (ACA) has the responsibility of validating attributes and in response it returns an Attribute Certificate (ACert) with the valid attribute values.
+
+ACA will have a database to hold attributes for each user and affiliation. This DB will be encrypted.
+
+1. userID _(The id passed by the user during enrollment)_
```

Comment:
These are the fields of the record, so I would suggest format them as clean as possible. No need to enclose the meaning in parenthesis and italicized. For example,  "userID: The id passed by the user during enrollment".  Also why some fields camel-case and others not? Be consistent. I would also suggest change `attribute key` to `attribute name` as `key` being used in cryptography.